fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/FabricLootSupplierBuilder.java

```diff
@@ -81,7 +81,7 @@ public FabricLootSupplierBuilder copyFrom(LootTable supplier) {
 	}
 
 	/**
-	 * Copies the pools and functions of the {@code supplier} to this builder.
+	 * Copies the pools and functions of the {@code supplier} to this builder.
```

Comment:
So a story on this one, apparently a symbol was there that prevents gradle from building on my arch setup, so the whitespace had to change

Reply:
Ah, I've probably made an accidental NBSP there (iirc that's my code?)

Reply:
Can checksyle fail on these? idk